docs/guide/docker.md

```diff
@@ -0,0 +1,114 @@
+---
+id: docker
+title: 🚢 Build docker image
+description: Build simple docker image that you can use on the production
+sidebar_position: 8
+---
+
+After we have completed the development of our application, the moment comes when we want to upload it to the real world, most often we use Docker for this, wrapping our application in a container.
+
+To do this we need to follow a few simple steps:
+
+## Step 1 - Setup project
+
+First create folder for our project run the following command in the console:
+
+```bash title="~/"
+mkdir example && cd example
+```
+
+Then we have to init go project:
+```bash title="~/example"
+go mod init example.org/demo
+```
+
+Add **Fiber** to our project:
+```bash title="~/example"
+go get github.com/gofiber/fiber/v2
+```
+
+## Step 2 - Hello world
+
+To start we shold create `main.go` file:
+```bash title="~/example"
+touch main.go
+```
+
+Next we should add following content to our file:
+
+```go title="~/app/main.go"
+package main
+
+import "github.com/gofiber/fiber/v2"
+
+func main() {
+	app := fiber.New()
+
+	app.Get("/", func(c *fiber.Ctx) error {
+		return c.SendString("Hello, World!")
+	})
+
+	app.Listen(":4000")
+}
+```
+
+## Step 3 - create Dockerfile
+
+Run the following command in the console:
+
+```bash
+touch Dockerfile
+```
+
+For the image we will use [Distroless](https://github.com/GoogleContainerTools/distroless) docker image, in the file we should add following content:
+
+```bash title="~/example/Dockerfile"
+FROM golang:1.21-alpine as base
+
+WORKDIR /app
+COPY . .
+
+RUN go mod download && go mod verify
+RUN GOOS=linux GOARCH=amd64 go build -o /server .
+
+FROM gcr.io/distroless/static-debian12
+COPY --from=base /server .
+
+ENV SOME_VAR=foo
+
+EXPOSE 4000
+CMD ["./server"]
+```
+
+:::info
+"Distroless" images contain only your application and its runtime dependencies. They do not contain package managers, shells or any other programs you would expect to find in a standard Linux distribution.
+:::
+
+:::tip
```

Comment:
This whole section is irrelevant to GoFiber. This is more for Docker docs

Reply:
Yes, that's right, but students often ask this question, why this particular image, and not, say, Docker or Ubuntu. Therefore, I decided to add this as an information section to immediately avoid possible questions in issue section.

But if it critical I can remove it.